elastic/kibana
# [SecuritySolutions] Create Entity Store 'entities/list' API

This PR introduces the following API routes for listing Entity Store "entities":

<meta charset="utf-8"><b style="font-weight:normal;" id="docs-internal-guid-9410c5d7-7fff-e873-6830-887939a306fb"><div dir="ltr" style="margin-left:-0.75pt;" align="left">
List Entities | GET /api/entity_store/entities/list
-- | --
</div></b>

The PR includes the following:
 - The OpenAPI schemas for the route
 - The actual Kibana side endpoint
 - Add searchEntities function to the `EntityStoreDataClient`
 

### How to test

1. Add some host/user data
    * Easiest is to use [elastic/security-data-generator](https://github.com/elastic/security-documents-generator)
2. Make sure to add `entityStoreEnabled` under `xpack.securitySolution.enableExperimental` in your `kibana.dev.yml`
3. In kibana dev tools or your terminal, call the `INIT` route for either `user` or `host`.
4. You should now see 2 transforms in kibana. Make sure to re-trigger them if needed so they process the documents.
5. Call the new API, and it should return entities 



Implements https://github.com/elastic/security-team/issues/10517

### Checklist
